clients/drive9-py/tests/test_e2e.py

```
@@ -0,0 +1,198 @@
+
+
+def _make_client() -> Client:
+    return Client(E2E_BASE, api_key=E2E_API_KEY)
+
+
+@pytest.fixture(scope="session")
+def client():
+    c = _make_client()
+    try:
+        c.list("/")
+    except Exception as exc:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)